definitions/procedures/pulp/remove.rb

```diff
@@ -0,0 +1,100 @@
+module Procedures::Pulp
+  class Remove < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove pulp2'
+      for_feature :pulp2
+    end
+
+    def sys
+      ForemanMaintain::Utils::SystemHelpers.new
+    end
+
+    def pulp_data_dirs
+      [
+        feature(:pulp2).data_dir + '/published',
+        feature(:pulp2).data_dir + '/content',
+        feature(:pulp2).data_dir + '/importers',
+        feature(:pulp2).data_dir + '/uploads'
+      ]
+    end
+
+    def pulp_packages
+      [
+        'pulp-server', 'python-pulp-streamer', 'pulp-puppet-plugins',
+        'python-pulp-rpm-common', 'python-pulp-common',
+        'pulp-selinux', 'python-pulp-oid_validation',
+        'python-pulp-puppet-common', 'python-pulp-repoauth',
+        'pulp-rpm-plugins'
+      ]
+    end
+
+    def mongo_packages
+      find_installed_packages('rh-mongodb34*').concat(
+        find_installed_packages('mongodb*')
+      ).uniq
+    end
+
+    def find_installed_packages(pattern)
+      cmd = "rpm -qa --queryformat '\n%{NAME} ' | grep '#{pattern}'"
```

Comment:
an rpm -qa is kinda expensive to do, it'd be a lot quicker to store the output and grep through it in ruby.  

Although i kinda question if we should actually check it or just provide some static yum remove comamnds?  I'm not opposed to checking it, just not sure if it will actually matter in the end

Reply:
The biggest reason I didn't create a static list is that the list of mongodb packages was enormous.

Reply:
oh i meant to literally:
```
yum remove "rh-mongodb34-*
```

that will cause yum to remove them all on its own. no need for us to figure it out

Reply:
D'oh! Silly me - of course!